utils/loggers/wandb/wandb_utils.py

```diff
@@ -43,6 +43,9 @@ def check_wandb_config_file(data_config_file):
 def check_wandb_dataset(data_file):
     is_trainset_wandb_artifact = False
     is_valset_wandb_artifact = False
+    if isinstance(data_file, dict):
+        # In that case another dataset manager has already processed it and we don't have to
+        return None
```

Comment:
it should return the data_file instead of None right?

Reply:
Yup, I think you're right. In this case, it would be None whenever both ClearML and Wandb are installed and a `clearml://` url is given, which would indeed crash. I will have to test this though.